conda_build/build.py

```diff
@@ -736,6 +736,30 @@ def copy_readme(m):
                   "as README.md and README.rst", file=sys.stderr)
 
 
+def jsonify_info_yamls(m):
+    iyd = "info_yaml.d"
+    ijd = "info_json.d"
+    src = join(dirname(m.meta_path), iyd)
+    res = []
+    if os.path.exists(src) and isdir(src):
+        for root, dirs, files in os.walk(src):
+            for file in files:
+                file = join(root, file)
+                bn, ext = os.path.splitext(os.path.basename(file))
+                if ext == '.yaml':
+                    dst = join(m.config.info_dir, ijd, bn+'.json')
+                    try:
+                        os.makedirs(os.path.dirname(dst))
+                    except:
```

Comment:
Use `except EnvironmentError` (equivalent to `except (IOError, OSError)`). If you want a catch-all, best practice is to always use `except Exception`. A naked `except` catches `KeyboardInterrupt` and several other exceptions that you really don't want to be catching.